src/components/PurchaseOrder/PO.js

```diff
@@ -274,6 +275,34 @@ const PO = ({
     [context, fetchOrder, order, orderErrorModalShow, sendCallout],
   );
 
+  const unopenOrder = useCallback(
+    () => {
+      const orderProps = {
+        workflowStatus: WORKFLOW_STATUS.pending,
+      };
+
+      toggleUnopenOrderModal();
+      setIsLoading(true);
+      updateOrderResource(order, mutator.orderDetails, orderProps)
+        .then(
+          () => {
+            sendCallout({
+              message: <SafeHTMLMessage id="ui-orders.order.unopen.success" values={{ orderNumber }} />,
+              type: 'success',
+            });
+
+            return fetchOrder();
```

Comment:
should the list be updates as well? if status is change, list might show previous value

Reply:
Added prop `refreshList` to clone, approve, open, unopen, close order